### PR TITLE
[NON-MODULAR] Command and Central Command Budgets

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

I mean, you can read the title can you not?

Adds Command and Central Command budgets. Captain, NT Rep, and Blueshield no longer drain Security for their paychecks, and the glorified Cargo Decks at the Titan now have a budget so they don't drain the actual Station's budget.

## Why It's Good For The Game

Security can now actually have a budget.

## Changelog
:cl:
add: Added Command and CC budgets
add: Added NT Rep, Captain and Blueshield to Command Budget
add: Added NT Admiral to CC budget.
/:cl:

### CODE REVIEW

**Comment by axietheaxolotl** [comment]
MY CHECKS NOOOOOOOOOOOOOO

File: code/modules/jobs/job_types/captain.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	plasmaman_outfit = /datum/outfit/plasmaman/captain
 
 	paycheck = PAYCHECK_COMMAND
-	paycheck_department = ACCOUNT_SEC
+	paycheck_department = ACCOUNT_CMD
```
Comment (GoldenAlpharex) [inline code/modules/jobs/job_types/captain.dm]:
```suggestion
	paycheck_department = ACCOUNT_CMD // SKYRAT EDIT - Original: paycheck_department = ACCOUNT_SEC
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Just add a little comment here to say that it was an edit.

On another note, we should probably do something with the proc that decides the price of an item so that people with the appropriate access get the appropriate price, because having to put all the Heads in the Command budget would feel a bit weird, just to fix one minor inconvenience, and would many more.

File: modular_skyrat/modules/blueshield/code/modules/jobs/job_types/blueshield.dm [hunk 2]
```diff
@@ -14,7 +14,7 @@
 	exp_granted_type = EXP_TYPE_CREW
 
 	paycheck = PAYCHECK_HARD
-	paycheck_department = ACCOUNT_SEC
+	paycheck_department = ACCOUNT_CMD
```
Comment (FlamingLily) [inline modular_skyrat/modules/blueshield/code/modules/jobs/job_types/blueshield.dm:17]: Personally, I would have the blueshield as a CC paycheck as well

**Comment by axietheaxolotl** [comment]
This got fucky with the shitty NTF merge, so I'm gonna close and re-open this to make my life infinitely easier.

**Pull request closed without merging**